tests/Event/Productions/ProductionCommandHandlerTest.php

```diff
@@ -97,4 +97,30 @@ public function itCannotAddAnEventThatAlreadyBelongsToAnotherProduction():void
             new AddEventToProduction($eventBelongingToFirstProduction, $secondProductionCommand->getProductionId())
         );
     }
+
+    /**
+     * @test
+     */
+    public function itCanRemoveAnEventFromAProduction(): void
```

Comment:
Could you switch it to snake_case to be consistent with other tests (or is there a reason for using camelCase here specifically?)

Reply:
No specific reason other than muscle memory. I'll snake case the tests here

Reply:
You know what, I'll do it in the follow-up PR because that one adds another test.

Reply:
Yeah ofc, it's no biggie :)

Reply:
Fixed here: https://github.com/cultuurnet/udb3-silex/pull/465/commits/72912f14550753adf5cdc2168ebc2a4c52c3be82